go.mod

```diff
@@ -97,7 +97,7 @@ require (
 	github.com/IBM/sarama v1.43.3
 	github.com/Jeffail/gabs/v2 v2.7.0
 	github.com/TykTechnologies/graphql-go-tools/v2 v2.0.0-20240509085643-e95cdc317e1d
-	github.com/TykTechnologies/kin-openapi v0.90.0
+	github.com/TykTechnologies/kin-openapi v0.90.1-0.20250416082231-59815ff298a7
```

Comment:
@olamilekan000 the plan is to completely rely on the fork for now.
we're still importing upstream v0.0.115 https://github.com/TykTechnologies/tyk/pull/7022/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R42

a replace directive might be required to use the fork
github.com/getkin/kin-openapi => github.com/TykTechnologies/kin-openapi v0.90.1-0.20250416082231-59815ff298a7